### ISSUE TITLE
Help page

### ISSUE THREAD

**Issue by hao-fang** [acl-org/acl-2020-virtual-conference#20]
**Number of Volunteers**: 1

- [ ] Go through the ICLR2020 FAQ [here](https://iclr.cc/virtual_2020/about.html) and see if we can re-use any of them.
- [ ] post here (or RocketChat #website-volunteers) channel to request for more information.

**Comment by Robert-Alonso** [comment]
Hi @hao-fang, I was going through the ICLR 2020 FAQ and identified the following points, which may be part of the info we’ll need for ACL:
- There is an option to export the schedule info as an ics file, this may be a feature for https://github.com/acl-org/acl-2020-virtual-conference/issues/18 and an initial alternative for https://github.com/acl-org/acl-2020-virtual-conference/issues/31
- 6Connex was used in addition to Rocket.Chat, Zoom and SlidesLive, specifically for interaction with sponsors
- There is a #helpdesk channel in Rocket.Chat, which I see will be coordinated in https://github.com/acl-org/acl-2020-virtual-conference/issues/62 
- There are instructions for registration in Rocket.Chat which include a backup invite link besides the automatic account creation info sent by email
- There is a [Code of Conduct](https://iclr.cc/public/CodeOfConduct) 
- There is a [Privacy Policy](https://iclr.cc/public/PrivacyPolicy)

**Comment by hao-fang** [comment]
@Robert-Alonso Thanks for looking into this!

> * There is an option to export the schedule info as an ics file, this may be a feature for #18 and an initial alternative for #31

Yes, as a basic version, we will follow ICLR to generate all the ics files. I'm not sure how it works. This will be added to regular paper pages #16 as each live QA session will get an ics file. Can you add this to our Help first?

> * 6Connex was used in addition to Rocket.Chat, Zoom and SlidesLive, specifically for interaction with sponsors

We will not use 6connex for ACL2020. We are building our own sponsor experience #25. Can you follow up there to make sure they will prepare a FAQ about virtual sponsor booth experience? I think they just started the work so you may need to wait for a couple of days.

> * There is a #helpdesk channel in Rocket.Chat, which I see will be coordinated in #62

Yes. We already have a #helpdesk channel. Can you add this to our Help page first?

> * There are instructions for registration in Rocket.Chat which include a backup invite link besides the automatic account creation info sent by email

I'm not sure about this one. I'll check with program chairs on this. Can you make sure to follow up with me if I do not get back to you next week?

> * There is a [Code of Conduct](https://iclr.cc/public/CodeOfConduct)
> * There is a [Privacy Policy](https://iclr.cc/public/PrivacyPolicy)

Let's add these two first and I can ask the committee to review it.

**Comment by dreasysnail** [comment]
Hi @Robert-Alonso
Another volunteer, Prabhat, wants to work on the social page. Do you mind to first sign up with the Help page?

**Comment by Robert-Alonso** [comment]
Sure, I'll be working on the help page first then

**Comment by hao-fang** [comment]
@Robert-Alonso @dreasysnail -- It seems that this issue is done? I'm closing it now.
But if you are still working on it, feel free to re-open or create a new issue.

**Comment by Robert-Alonso** [comment]
Hi @hao-fang, ok 👍 the remaining info to update would be just the details about last minute registration and browser compatibility, which depend on [#55](https://github.com/acl-org/acl-2020-virtual-conference/issues/55) and [#90](https://github.com/acl-org/acl-2020-virtual-conference/issues/90).
